packages/integrations/image/README.md

```diff
@@ -153,7 +334,7 @@ const imageUrl = 'https://www.google.com/images/branding/googlelogo/2x/googlelog
   
   The `<Image />` component can also be used to optimize images in markdown pages. For local images imported from your project's `src` directory, use Astro's the `setup` frontmatter to import the image file.
```

Comment:
@tony-sull - We should probably remove this section "Images in Markdown" entirely now?  You could choose to replace it with an MDX example.

Reply:
Yep, great call!  I'll remove that now and can revisit for an MDX example once that's been tested a bit more